### ISSUE TITLE
[Bug]: Forking selected pages from template from Add new page popover is crashing the app

### ISSUE THREAD

**Issue by Parthvi12** [appsmithorg/appsmith#17272]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

On forking selected pages from Add new page popover, crashes the app with page not found error

<img width="1440" alt="Screenshot 2022-10-03 at 4 03 00 PM" src="https://user-images.githubusercontent.com/80334441/193557237-f4e61338-55f7-41b2-82c3-2fa654ab0d65.png">

### Steps To Reproduce

1. Click on + on left pane of Pages
2. Select Add Page from Template option
3. Select 'Search' page of Customer Support Dashboard template 
Note : Happening on unchecking first page, and selecting other pages.
5. Observe the app crashes

### Version

Release

**Comment by Parthvi12** [comment]
@akash-codemonk Can you try selecting only "Search" page and fork, it crashes for me

**Comment by akash-codemonk** [comment]
Yeah tried that I see resource not found error as mention in https://github.com/appsmithorg/appsmith/issues/17270

**Comment by Parthvi12** [comment]
@akash-codemonk Yes but there we are forking from start from template card and on refresh the app works fine, but here we are forking app from add new page popover and app goes in bad state, even on hard refresh it is not recovered. Hence marked as release blocker.

**Comment by Richarex** [comment]
This is fixed now forking selected pages from template is not crashing now. https://www.loom.com/share/25ef55c41809485f93c4faacbd5abbda

**Comment by Parthvi12** [comment]
Reopening as it's happening on release @akash-codemonk 
<img width="1440" alt="Screenshot 2022-10-11 at 2 43 38 PM" src="https://user-images.githubusercontent.com/80334441/195049636-7b76e99c-e1ee-492d-a1e2-b9e8ef7cfa3d.png">

**Comment by akash-codemonk** [comment]
same as https://github.com/appsmithorg/appsmith/issues/17383? Did you unselect the first page?

**Comment by AnaghHegde** [comment]
@Parthvi12 Raised a PR to fix issue - https://github.com/appsmithorg/appsmith/pull/17474

**Comment by Richarex** [comment]
This is fixed now verified it on release.
